main.tf

```diff
@@ -4,16 +4,24 @@
 # Configures standard account & IAM parameters
 ##############################################################################
 
+locals {
+  # If shell_settings_enabled is true, then skip_cloud_shell_calls must be false.
+  cloud_shell_settings_validate_condition = var.shell_settings_enabled == true && var.skip_cloud_shell_calls == true
```

Comment:
@rajatagarwal-ibm why are we doing this validation at all? If `skip_cloud_shell_calls` is set to true, then it doesn't matter what the value of `shell_settings_enabled` is because you have just told terraform to skip checking and setting cloud shell settings

Reply:
perhaps update the variable description of `shell_settings_enabled` to make it clear that if `skip_cloud_shell_calls` is true, the setting is ignored